VEDA/utils.py

```diff
@@ -8,8 +8,9 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
-DEFAULT_CONFIG_FILE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+CONFIG_FILE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
CONFIG_ROOT_DIR would be better? As currently, its giving an impression as if it is config file path but its config dir path.